client/src/components/CardDialog/index.ts

```diff
@@ -0,0 +1,9 @@
+export { default as DialogItem } from './DialogItem/DialogItem';
+export { default as dialogItemStyles } from './DialogItem/dialogItemStyles';
+export { default as DialogItemGroup } from './DialogItemGroup/DialogItemGroup';
+export { default as CardDialog } from './CardDialog';
+export { default as cardDialogStyles } from './cardDialogStyles';
+export { default as DatePicker } from './DatePicker/DatePicker';
+export { default as datePickerStyles } from './DatePicker/datePickerStyles';
+export { cardDialogItems } from './initialDialogData';
+export { default as DialogActionButton } from './DialogActionButton/DialogActionButton';
```

Comment:
I like this `default as ...` syntax better than what I've been doing; let's stick to it but change line 8 to keep consistent

Reply:
Done